geoh5py/ui_json/input_file.py

```diff
@@ -166,7 +165,9 @@ def validations(self, valid_dict: dict | None):
     def validators(self):
```

Comment:
I would propose to rename this to `input_validation` and `ui_validation` instead of `validators` and `ui_validators`. Just so we make a distincation with `validators` of the `InputValidation` class.

Reply:
Let's talk through the naming, I have a different opinion..